chain/epoch_manager/src/lib.rs

```diff
@@ -487,10 +487,7 @@ impl EpochManager {
     pub fn compute_stake_return_info(
```

Comment:
Add comment what is returned. When the return type is not named, we need to be always descriptive